README.md

```diff
@@ -1,93 +1,417 @@
-# Okta Management SDK for Golang
-[![Build Status](https://img.shields.io/travis/okta/okta-sdk-golang/master.svg)](https://travis-ci.org/okta/okta-sdk-golang)
+[<img src="https://devforum.okta.com/uploads/oktadev/original/1X/bf54a16b5fda189e4ad2706fb57cbb7a1e5b8deb.png" align="right" width="256px"/>](https://devforum.okta.com/)
+[![Build Status](https://img.shields.io/travis/okta/okta-sdk-golang.svg?logo=travis)](https://travis-ci.org/okta/okta-sdk-golang)
+[![GitHub](https://img.shields.io/github/license/okta/okta-sdk-github.svg)](https://github.com/okta/okta-sdk-golang)
```

Comment:
Remove `/license/` 🙂

Reply:
thoughts on why?

Reply:
nevermind... realize it is doubled there